### PR TITLE
Korean: add missing translations

### PR DESCRIPTION
**Pull request by crwusiz**
translation unfinished fix

### CODE REVIEW

**Comment by sshane** [comment]
Thanks! Do you have anyone you know or a friend to review the translations?

File: selfdrive/ui/translations/main_ko.ts [hunk 1]
```diff
@@ -479,7 +479,7 @@ location set</source>
         <location filename="../qt/offroad/networking.cc" line="87"/>
         <location filename="../qt/offroad/networking.cc" line="97"/>
         <source>for &quot;%1&quot;</source>
-        <translation>하기위한 &quot;%1&quot;</translation>
+        <translation>&quot;%1&quot;에 접속하려면 인증이 필요합니다</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_ko.ts:482]: The new translation adds more text than "for"? Password is already translated right above

**Comment by sshane** [comment]
Let's make this PR strictly add the missing translations (e2e long), adding/fixing a bunch of other translations makes it harder to review and merge this PR

File: selfdrive/ui/translations/main_ko.ts (hunk 1 above)
Comment (crwusiz) [reply selfdrive/ui/translations/main_ko.ts:482]: The additional translation is better because it is about accessing the wifi ssid

**Comment by crwusiz** [comment]
> Let's make this PR strictly add the missing translations (e2e long), adding/fixing a bunch of other translations makes it harder to review and merge this PR. @crwusiz can you remove other additions and move those to another PR?

Okay.. I will upload the translation modification except e2e long to another pr.

**Pull request merged**